kolibri/plugins/management/assets/src/vue/data-page/index.vue

```diff
@@ -1,16 +1,40 @@
 <template>
 
-  <div>
+  <div class="wrapper">
+
     <h1>Export Usage Data</h1>
 
     <p>
-      Download a comma-separated spreadsheet containing information about users and
+      Download comma-separated spreadsheets containing information about users and
       their interactions with the content on this device.
     </p>
```

Comment:
A more user oriented language will sound kinda like "To analyse user interactions with the contents, you can download all the interaction tracking data in CSV format."